phantomgames/Constants/Constants.swift

```diff
@@ -24,9 +24,12 @@ struct Constants {
         static let searchBarPlaceholder = "Search Game title"
     }
     struct Endpoints {
-        static let search = "https://www.freetogame.com/api/games"
-        static let home = "https://www.freetogame.com/api/games?sort-by=alphabetical"
+        static let search = "https://www.freetogame.com/api/games?sort-by=alphabetical"
+        static let homeForCollectionView = "https://www.freetogame.com/api/games?sort-by=release-date"
+        static let homeForTableView = "https://www.freetogame.com/api/games"
         static let gameDetail = "https://www.freetogame.com/api/game?id="
+        static let pcGamesURL = "https://www.freetogame.com/api/games?platform=pc"
+        static let browserGamesURL = "https://www.freetogame.com/api/games?platform=browser"
     }
```

Comment:
Please put a new line between all of these structs. It is a swiftlint warning. Please do a pr to clen up all swiftlint warnings :)

Reply:
I did not have ny swift lint warning on this file